tests/transports-governance/test_runner_hierarchical_comprehensive.py

```
@@ -0,0 +1,830 @@
+            return TestSuiteResult(
+                name=suite_name,
+                status="error",
+                duration=duration,
+                passed=0,
+                failed=0,
+                skipped=0,
+                errors=1,
+                details={"error": str(e)},
+            )
+
+    def _parse_pytest_output(
```

Ruff (0.12.2)

`_parse_pytest_output` is too complex (11 > 10)

(C901)